test/jdk/java/util/concurrent/tck/CompletableFutureTest.java

```diff
@@ -80,7 +80,14 @@ public static Test suite() {
         return new TestSuite(CompletableFutureTest.class);
     }
 
-    static class CFException extends RuntimeException {}
+    static class CFException extends RuntimeException {
+        // This makes sure that CompletableFuture still behaves appropriately
+        // even if thrown exceptions end up throwing exceptions from their String
+        // representations.
+        @Override public String getMessage() {
```

Comment:
I decided to do this to make sure that *all pre-existing test cases* would still work even if the exception used has a misbehaving toString() (toString() transitively calls getLocalizedMessage() which in turn calls getMessage())